cupy/core/_scalar.pyx

```diff
@@ -37,7 +62,7 @@ cdef dict _dtype_kind_size_dict = {}
 
 cdef _setup_type_dict():
     cdef char k
-    for i in _dtype.all_type_chars:
+    for i in _dtype_module.all_type_chars:
```

Comment:
Is it better to change `all_types_chars` to `cdef` to avoid `_dtype_module`? (correct me if I'm wrong)